OloEngine/src/OloEngine/Networking/Prediction/NetworkInputBuffer.cpp

```
@@ -0,0 +1,95 @@
+#include "OloEnginePCH.h"
+#include "NetworkInputBuffer.h"
+
+namespace OloEngine
+{
+    NetworkInputBuffer::NetworkInputBuffer(u32 capacity)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_Entries

(cppcoreguidelines-pro-type-member-init)